GITHUB_API_NUANCE.md

```diff
@@ -0,0 +1,80 @@
+# Nuance in the GitHub API with Probot
```

Comment:
Nit: I feel like `github-access-tokens.md` or something similar might be clearer than `GITHUB_API_NUANCE.md`; similar for the title, highlighting that it's more about tokens than the API

Reply:
Done